<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGlobal</class>
 <widget class="QDialog" name="SettingsGlobal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="settingsGlobalTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>521</width>
     <height>393</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>391</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalWidget">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>343</width>
       <height>297</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="beforeCheckBox">
        <property name="text">
         <string>Prompt before closing tabs?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overwriteCheckBox">
        <property name="text">
         <string>Prompt if new search overwrites previous results?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cancelCheckBox">
        <property name="text">
         <string>Prompt if cancel pressed during search?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="closingCheckBox">
        <property name="text">
         <string>Prompt before closing Searchmonkey?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lastCheckBox">
        <property name="text">
         <string>Close Searchmonkey if last tab deleted?</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="restoreCheckBox">
        <property name="text">
         <string>Restore previous state after restart?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="newCheckBox">
        <property name="text">
         <string>Always start new search in a new tab?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="busyCheckBox">
          <property name="text">
           <string>Automatically start search in new tab if current tab busy?</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="accurateCheckBox">
        <property name="text">
         <string>Show accurate progress bar (slower to find results)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="syntaxComboBox">
        <item>
         <property name="text">
          <string>Use Regular Expression Syntax (Perlesque)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Use Extended Regular Expression Syntax (greedy capture)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wildcard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreCheckBox">
        <property name="text">
         <string>Ignore file open errors (best for unattended searching)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restorePushButton">
        <property name="text">
         <string>Restore factory defaults</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="lookWidget">
    <attribute name="title">
     <string>Look and Feel</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="editorLabel">
        <property name="text">
         <string>Default text editor:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="editorPushButton">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="linenoCheckBox">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>21</y>
       <width>163</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show line numbers in results?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="whitespaceCheckBox">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>44</y>
       <width>199</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Trim whitespace from matching lines?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="matchFontPushButton">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>67</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set match font...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="headingFontPushButton">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>96</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set heading font...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lookTextEdit">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>212</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>71</height>
      </size>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; font-style:italic;&quot;&gt;Example results (27&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; font-style:italic; vertical-align:super;&quot;&gt;rd&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; font-style:italic;&quot;&gt; May 2010)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600; text-decoration: underline;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;10&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - This box shows how matching &lt;/span&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#0000ff;&quot;&gt;results&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; will look&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;14&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; - amidst the standard text which does not have &lt;/span&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#5555ff;&quot;&gt;results&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; shown&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="matchesLabel2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[0: unlimited]</string>
     </property>
    </widget>
    <widget class="QLabel" name="matchesLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>78</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Matching files:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="matchesSpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>159</width>
       <height>20</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="editorLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>notepad.exe</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="presetsWidget">
    <attribute name="title">
     <string>Presets</string>
    </attribute>
    <widget class="QListWidget" name="presetListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>301</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="presetAddButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="presetDeleteButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="presetDownButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Down</string>
     </property>
    </widget>
    <widget class="QLabel" name="presetLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Preset file modes:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="presetEditButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="presetUpButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Up</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="settingsGlobalButtonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
